### PR TITLE
[TW-1160] Refresh "Using SOAP Requests" doc

### PR DESCRIPTION
**Pull request by nerminamiller-postman**
Updates screenshots and adds small copyedits

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
One super minor comment, otherwise LGTM! 🌟

File: src/pages/docs/sending-requests/soap/making-soap-requests.md [hunk 1]
```diff
@@ -20,80 +20,65 @@ contextual_links:
...
 ## Adding body data
 
 1. In the __Body__ tab, select __raw__ and choose __XML__ from the dropdown list.
-1. Enter your XML in the text entry area.
 
-    ![SOAP body type](https://assets.postman.com/postman-docs/v10/soap-body-type-v10.jpg)
+    ![SOAP body type](https://assets.postman.com/postman-docs/v10/soap-body-type-v10-16.jpg)
 
-If you want to test the number conversion SOAP API used in the last section, enter the following XML in the text entry area:
+1. Enter your XML in the text entry area. To test the number conversion SOAP API used in the last section, enter the following XML in the text entry area:
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/sending-requests/soap/making-soap-requests.md:52]: Instead of "last", what do you think about using "previous" instead?

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

**Review by claireriley — APPROVED** [review]
LGTM! ⭐

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/sending-requests/soap/making-soap-requests.md (hunk 1 above)
Comment (nerminamiller-postman) [reply src/pages/docs/sending-requests/soap/making-soap-requests.md]: Done! Thank you!

**Pull request merged**